app/services/estate_sso_mapper.rb

```diff
@@ -0,0 +1,56 @@
+class EstateSSOMapper
+  DIGITAL_ORG = 'digital.noms.moj'.freeze
+
+  def self.grouped_estates
+    @grouped_estates ||= begin
+      grouped_estates = Hash.new { |h, k| h[k] = [] }
+      Estate.where.not(group: nil).each do |estate|
+        grouped_estates[estate.group] << estate.sso_organisation_name
+      end
+      grouped_estates.values.freeze
+      grouped_estates.freeze
+    end
+  end
+
+  def self.reset_grouped_estates
```

Comment:
Yeah, I prefer to avoid generally. The issue here is that the order of the tests matter, so if this spec file runs before the feature specs, then some of those fail, so the best thing is to always cleanup after using the method